doc/HLD_of_Motr_Object_Index.rst

```diff
@@ -0,0 +1,338 @@
+=========================================
+High level design of a Motr Object Index
```

Comment:
s
/High level design of a Motr Object Index
/High Level Design of a Motr Object Index

Reply:
Let's add a "OBSOLETED" at the end of this title.
Some of the detailed implementation are out-dated. But the basic idea is still valid.

Reply:
Comment Incorporated.